dev/setup-fleet

```diff
@@ -3,14 +3,16 @@
 
 set -euxo pipefail
 
+cluster_name=${1-upstream}
```

Comment:
Other scripts use FLEET_E2E_CLUSTER=k3d-upstream

Reply:
I followed `dev/setup-k3d`'s example. Do we want to align all scripts on this?

Reply:
Ah, I misread. It's the first argument, not an env var. 
No, I don't want to align the scripts, that would break everyone's setup. But if we do, "cluster" 
 should really be called "context" :)